src/Symfony/Component/Messenger/Bridge/Kafka/Transport/KafkaTransport.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\Kafka\Transport;
+
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Transport\Serialization\PhpSerializer;
+use Symfony\Component\Messenger\Transport\Serialization\SerializerInterface;
+use Symfony\Component\Messenger\Transport\TransportInterface;
+
+class KafkaTransport implements TransportInterface
+{
+    private KafkaReceiver $receiver;
+    private KafkaSender $sender;
```

Comment:
these should also use the proper interfaces respectively instead of the concrete implementations, to allow for decoration.